addons/website_sale_autocomplete/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Google places autocompletion',
+    'category': 'Website/Website',
+    'summary': 'Assist your users with automatic completion & suggestions when filling forms',
+    'version': '1.0',
+    'description': "Assist your users with automatic completion & suggestions when filling forms",
```

Comment:
This may be misleading:
```suggestion
    'summary': 'Assist your users with automatic completion & suggestions when filling their address during checkout',
    'version': '1.0',
    'description': "Assist your users with automatic completion & suggestions when filling their address during checkout",
```